src/models.rs

```diff
@@ -1131,3 +1134,167 @@ impl Metadata {
 
 /// Represent a String path instead of passing a PathBuf struct
 pub type StringPath = String;
+
+pub trait ToTerraformString {
+    fn to_terraform_format_string(&self) -> String;
+}
+
+pub trait ToHelmString {
+    fn to_helm_format_string(&self) -> String;
+}
+
+/// Represents a domain, just plain domain, no protocol.
+/// eq. `test.com`, `sub.test.com`
+#[derive(Clone)]
+pub struct Domain {
+    raw: String,
+    top_domain: String,
```

Comment:
OPT: call it root domain to be consistent ?

Reply:
Yep, you are right, thanks :)